File: ts/src/special-cards/supply-drop.ts
import { SpecialCardDefinition } from './types';

export const supplyDropCard: SpecialCardDefinition = {
  id: 'supply-drop',
  name: 'Supply Drop',
  shortName: 'Supply',
  description: 'Passive: start each level with +1 shuffle and +1 deal.',
  rarity: 'common',
  cost: 7,
  accent: '#ff8b8b',
  bonusDealsPerLevel: 1,
  bonusShufflesPerLevel: 1
};

File: godot/Scripts/SpecialCards/SupplyDrop.cs
namespace Roshambo.SpecialCards;

public static class SupplyDrop
{
	public static readonly SpecialCardDefinition Definition = new()
	{
		Id = "special:supply-drop",
		Name = "Supply Drop",
		ShortName = "Supply",
		Description = "Each level: +1 shuffle, +1 deal.",
		Rarity = SpecialCardRarity.Rare,
		Cost = 10,
		Accent = "#2196f3",
		BonusDealsPerLevel = 1,
		BonusShufflesPerLevel = 1
	};
}

